app/(auth)/sign-up/[[...sign-up]]/page.tsx

```diff
@@ -3,8 +3,10 @@ import { SignUp } from '@clerk/nextjs'
 import { type Metadata } from 'next'
 
 export const metadata: Metadata = {
+  metadataBase: new URL(`https://${process.env.NEXT_PUBLIC_VERCEL_URL}` ?? 'http://localhost:3000'),
```

Comment:
Same issue.

Reply:
Should be sorted now. Please can you take another look?